src/ua/ithillel/lesson6/main/java/MethodsClass.java

```diff
@@ -0,0 +1,32 @@
+import java.util.*;
+import java.util.stream.Collectors;
+
+public class MethodsClass {
+    public static int countOccurance(List<String> wordsList, String word) {
+        return (int) wordsList.stream().filter(el -> el.equals(word)).count();
```

Comment:
```suggestion
        return (int) wordsList.stream().filter(el -> Objects.equals(el, word)).count();
```
лише для інформації, як механізм захисту від null-ів
https://docs.oracle.com/javase/8/docs/api/java/util/Objects.html#equals-java.lang.Object-java.lang.Object-